Makefile

```diff
@@ -5,6 +5,7 @@
 .PHONY: build build-ios build-android unsigned-ios unsigned-android
 .PHONY: build-pr can-build-pr prepare-pr
 .PHONY: test help
+.PHONY: fastlane
```

Comment:
Minor nit, but it seems weird to have the Makefile have a phony `fastlane` target when there's an actual folder called `fastlane`. Is there perhaps some file we can look for to tell if fastlane has actually been installed?